AK/FixedArray.h

```diff
@@ -160,6 +160,12 @@ class FixedArray {
         ::swap(m_elements, other.m_elements);
     }
 
+    void fill_with(T value)
```

Comment:
can we make this a const reference to avoid the one pointless copy?

Reply:
godbolt says this is optimized to vectorized memcopy, but sure